spec/game_runner_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/game_runner.rb'
```

Comment:
You don't need to have the `.rb` at the end of required ruby files. It's a good convention to leave the extension off in these cases.

Reply:
✅